diff --git a/torchlibrosa/stft.py b/torchlibrosa/stft.py
index 4ba00c2..01b8470 100644
--- a/torchlibrosa/stft.py
+++ b/torchlibrosa/stft.py
@@ -190,7 +190,7 @@ class STFT(DFTBase):
         fft_window = librosa.filters.get_window(window, self.win_length, fftbins=True)
 
         # Pad the window out to n_fft size.
-        fft_window = librosa.util.pad_center(fft_window, n_fft)
+        fft_window = librosa.util.pad_center(fft_window, size=n_fft)
 
         # DFT & IDFT matrix.
         self.W = self.dft_matrix(n_fft)
