diff --git a/piano_transcription_inference/utilities.py b/piano_transcription_inference/utilities.py
index 75efa37..eff3a9c 100644
--- a/piano_transcription_inference/utilities.py
+++ b/piano_transcription_inference/utilities.py
@@ -553,7 +553,7 @@ def load_audio(path, sr=22050, mono=True, offset=0.0, duration=None,
                 y = librosa.core.audio.to_mono(y)
 
         if sr is not None:
-            y = librosa.core.audio.resample(y, sr_native, sr, res_type=res_type)
+            y = librosa.core.audio.resample(y, orig_sr=sr_native, target_sr=sr, res_type=res_type)
 
         else:
             sr = sr_native
